packages/components/src/FilterBar/FilterBar.tsx

```diff
@@ -27,13 +28,19 @@ export const FilterBar = <ValuesMap extends FiltersValues>({
   <FilterBarProvider<ValuesMap> filters={filters} {...providerProps}>
     {(activeFilters): JSX.Element => (
       <div className={classnames(styles.filterBar, classNameOverride)}>
-        {Object.values(activeFilters).map(({ id, Component }) => (
-          // `id` will always be `string`, but keyof ValuesMap transformed it
-          <React.Fragment key={id as string}>
-            {React.cloneElement(Component, { id })}
-          </React.Fragment>
-        ))}
-        <AddFiltersMenu />
+        <div className={styles.filtersContainer}>
```

Comment:
Might need to become a live region to announce changes in the bar?

Reply:
Hmm... that's an interesting one. A polite one?

Reply:
On second thought let's not complicate the screen reader experience on a hunch, perhaps the button's "Clear all filters" description is enough.